src/test/java/api/support/fixtures/FeeFineFixture.java

```diff
@@ -0,0 +1,32 @@
+package api.support.fixtures;
+
+import org.folio.circulation.domain.FeeFine;
+import org.folio.circulation.support.http.client.IndividualResource;
+
+import api.support.builders.FeeFineBuilder;
+import api.support.http.ResourceClient;
+
+public final class FeeFineFixture {
```

Comment:
Are these really fee fine types?

Reply:
Yes, they are. The fee/fines domain has following terms:
* FeeFineOwner - owner of a fee/fine;
* FeeFine - this actually a fee/fine type: `Lost item fee`, `Lost item processing fee`, etc;
* Account - this is a fee/fine record for a user (charge).

Reply:
I think we need to increasingly consider that in the circulation context, some of these things might have different names.

I think we should explicitly include type in the fee / fine type name, even though the source domain does not (which really confuses me)

I think later we could consider that an account is actually a fee or a fine in this context.

I'm not sure about the owner, I think there is a better comment to continue that conversation.

Reply:
Ok, will rename fixture only for now, we can create a task to rename domain classes/services/repositories/etc. if it make sense.